jira_agile_metrics/calculators/impediments_test.py

```diff
@@ -175,6 +175,5 @@ def test_different_done_column(query_manager, settings, cycle_time_results):
 
     assert data.to_dict('records') == [
         {'key': 'A-2', 'status': 'Committed', 'flag': 'Impediment',     'start': _ts('2018-01-10'), 'end': _ts('2018-01-12')},
-        {'key': 'A-3', 'status': 'Done',      'flag': 'Impediment',     'start': _ts('2018-01-07'), 'end': _ts('2018-01-10')},
```

Comment:
Again, it makes me nervous that you've changed the expected test output without changing anything else here. Can you explain your thinking?

Reply:
This test is/was done based on the following config:
```
        'cycle': [
            {'name': 'Backlog',   'statuses': ['Backlog'],           'type': 'backlog'},
            {'name': 'Committed', 'statuses': ['Next'],              'type': 'accepted'},
            {'name': 'Build',     'statuses': ['Build'],             'type': 'accepted'},
            {'name': 'Test',      'statuses': ['Code review', 'QA'], 'type': 'accepted'},
            {'name': 'Done',      'statuses': ['Done'],              'type': 'complete'}
        ],
```
If `committed_column` is `Committed` and `done_column` is `Build`, then only the block should be reported:
```
{'start': date(2018, 1, 10), 'end': date(2018, 1, 12), 'status': 'Committed', 'flag': 'Impediment'},  # included
```